utils/multibase-script/multibase.py

```diff
@@ -0,0 +1,101 @@
+import os
```

Comment:
Is it possible to move these 3 files under this folder inside the below document as code sections and ask user to create these as files and then run ?

[Hosting-DID-Document.md](https://github.com/mosip/inji-certify/pull/145/files#diff-850ba716aa5053d32f41cc09270665fb6bf6ca9cbc07be88ea56498747641df0)

I know it will make it a little difficult for user, but currently we don't have a utils folder and we are introducing one with a python script (deviation from the current stack used in the repo) for this usecase. Which will become unnecessary in the next release of Inji certify, since we will be serving the DID document directly from Inji certify service.

Reply:
Hitesh and I had plans to dump some sample files and other scripts, schema-context.json examples, etc here. Should I move this to `docs/multibase-script` as this directory is supposed to just contain `docs`.

> a little difficult for user,

I've added relative path links to this script, it'll be easy to find but slightly harder to maintain(if we remove the script later on).